src/main/java/com/codegym/jrugotom5/repository/AdvertRepository.java

```diff
@@ -3,5 +3,9 @@
 import com.codegym.jrugotom5.entity.Advert;
 import org.springframework.data.repository.CrudRepository;
 
+import java.util.Date;
+import java.util.List;
+
 public interface AdvertRepository extends CrudRepository<Advert, Long> {
+    List<Advert> findAllByCreatedDateBetween(Date from, Date to);
```

Comment:
Better would be to use LocalDate, not Date. Because in Advert entity we have LocalDate

Reply:
done